packages/ccip-svm/src/encoders/svmExtraArgsV1.ts

```diff
@@ -0,0 +1,131 @@
+import { address, getAddressEncoder, getAddressDecoder, getAddressFromPublicKey } from '@solana/kit'
+
+/**
+ * SVM Extra Args V1 tag: bytes4(keccak256("CCIP SVMExtraArgsV1"))
+ */
+export const SVM_EXTRA_ARGS_V1_TAG = 0x1f3b3aba
+
+export type Address = ReturnType<typeof address>
+
+export interface SVMExtraArgsV1 {
+    computeUnits?: number
+    accountIsWritableBitmap?: bigint
+    allowOutOfOrderExecution?: boolean
+    tokenReceiver?: Address
+    accounts?: Address[]
+}
+
+/**
+ * Creates properly encoded extraArgs buffer for SVM destinations
```

Comment:
Consider linking to docs or otherwise putting in some details into how the extraArgs are computed? many users of the SDK will not fully understand the operations within the implementation of the function so would help them to know where to 'study up' on this?

Reply:
@andrejrakic to not be brittle for future versions should re link to one level higher in the docs tree? docs urls may change too

https://docs.chain.link/ccip/api-reference/svm  instead of /v0.1.1 etc?